lightning-network-tools/taproot-assets/rfq.md

```diff
@@ -8,6 +8,8 @@ description: >-
 
 When sending Taproot Assets over the Lightning Network, an Edge Node is needed. This Edge Node receives the Taproot Asset in the channel with the direct user and swaps it for bitcoin. As the swap rate between Taproot Assets and bitcoin likely fluctuates, the user may make a Request for Quote to the Edge Node before generating a Lightning Network invoice or initiating a payment.
 
+This request is made using BOLT 01 messages over an existing encrypted and authenticated BOLT 08 connection. The connection already exists because it is used for establishing and coordinating the Taproot Asset Channel in a way very similiar to normal Lightning channels. For more information about connection and messaging, see the [Last Mile Routing](https://github.com/Roasbeef/blips/blob/tap-blip/blip-tap.md#last-mile-routing) section of [Taproot Asset Channels bLIP](https://github.com/Roasbeef/blips/blob/tap-blip/blip-tap.md).
```

Comment:
nit: could insert links to the spec? https://github.com/lightning/bolts/blob/master/01-messaging.md

Reply:
Would also be great to link to this document: https://github.com/lightninglabs/taproot-assets/blob/main/docs/rfq-and-decimal-display.md

I've created a [PR to sync the `tapd` docs automatically to this repo](https://github.com/lightninglabs/docs.lightning.engineering/pull/717), so the file should soon be available under `docs/taproot-assets/rfq-and-decimal-display.md`.

Reply:
> nit: could insert links to the spec? https://github.com/lightning/bolts/blob/master/01-messaging.md

You mean in `rfq.md` or `blip-tap.md`?

Reply:
> Would also be great to link to this document: https://github.com/lightninglabs/taproot-assets/blob/main/docs/rfq-and-decimal-display.md
> 
> I've created a [PR to sync the `tapd` docs automatically to this repo](https://github.com/lightninglabs/docs.lightning.engineering/pull/717), so the file should soon be available under `docs/taproot-assets/rfq-and-decimal-display.md`.

So will `rfq-and-decimal-display.md` be rendered by the gitbook, or do you just want to link into this github repository?

Reply:
> You mean in rfq.md or blip-tap.md?

I mean to link the words `BOLT 01` and `BOLT 08` to their definitions.

> So will rfq-and-decimal-display.md be rendered by the gitbook, 

Yes, once it's synced to this repo, it will be rendered and just needs to be linked.

Reply:
The `rfq-and-decimal-display.md` document is now available: https://github.com/lightninglabs/docs.lightning.engineering/blob/master/docs/taproot-assets/rfq-and-decimal-display.md

Reply:
> The `rfq-and-decimal-display.md` document is now available: https://github.com/lightninglabs/docs.lightning.engineering/blob/master/docs/taproot-assets/rfq-and-decimal-display.md

How do I link to it? I tried https://docs.lightning.engineering/the-lightning-network/taproot-assets/rfq-and-decimal-display but that does not work.

Reply:
> > You mean in rfq.md or blip-tap.md?
> 
> I mean to link the words `BOLT 01` and `BOLT 08` to their definitions.

Done

Reply:
Hmm, good question. I thought it would just be https://docs.lightning.engineering/docs/taproot-assets/rfq-and-decimal-display.
But not sure how that works with synced documents. Can you for now please just add https://github.com/lightninglabs/taproot-assets/blob/main/docs/rfq-and-decimal-display.md, since it's such an important document for `v0.5.0` and onward.

cc @Liongrass: how can we link to docs that are synced by the cronjob from other repos?

Reply:
Yes, we are able to link to synced posts without an issue, given they are synced into the `docs.lightning.engineering` repo

Reply:
I tried. How would we link to this? https://github.com/lightninglabs/docs.lightning.engineering/blob/master/docs/taproot-assets/rfq-and-decimal-display.md

I tried both https://docs.lightning.engineering/docs/taproot-assets/rfq-and-decimal-display and https://docs.lightning.engineering/docs/taproot-assets/rfq-and-decimal-display.md but that didn't work... Is there another prefix I need to use?

Reply:
I believe the document needs to be linked from the `summary.md`: https://github.com/lightninglabs/docs.lightning.engineering/blob/master/SUMMARY.md 
Under `* [Taproot Assets](lightning-network-tools/taproot-assets/README.md)` it would need `  * [Asset Decimal Display](/docs/taproot-assets/rfq-and-decimal-display.md)`

Reply:
Then the document would show up in the Builder's Guide as a separate guide, and can be linked from other guides

Reply:
Thanks @Liongrass!
@ZZiigguurraatt it's now available under https://docs.lightning.engineering/lightning-network-tools/taproot-assets/rfq-and-decimal-display.